www/source/docs/using-builder.html.md.erb

```diff
@@ -26,12 +26,6 @@ description: Documentation for Chef Habitat uploading, auto-building, and deploy
 ---
 <%= partial "/partials/docs/using-builder-create-origin"%>
 ---
-<<<<<<< HEAD
-<%= partial "/partials/docs/using-builder-origin-keys"%>
----
-<%= partial "/partials/docs/using-builder-access-tokens"%>
----
-=======
```

Comment:
Do we not need either of these partials?  Seems odd that both would be removed to resolve a merge conflict.